test/framework/functional/output_http.go

```diff
@@ -0,0 +1,49 @@
+package functional
+
+import (
+	"strings"
+
+	log "github.com/ViaQ/logerr/v2/log/static"
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/runtime"
+)
+
+const (
+	VectorImage        = "docker.io/timberio/vector:0.25.0-alpine"
```

Comment:
Prefer quay.io image if possible. Docker rate limits can cause failures if runing tests intensively.

Reply:
updated to use quay image

Reply:
@alanconway you mean sending to fluentd using http?

Reply:
updated PR to add a functional test which sends logs from `vector` to `fluentd` using `http`